### ISSUE TITLE
Broken link for Perkakas

### ISSUE THREAD

**Issue by dhiemaz** [kitabisa/buroq#13]
Perkakas link is broken 

https://github.com/kitabisa/perkakas

**Comment by ardityawahyu** [comment]
@dhiemaz 
yes, as the repo now in private. Will update it to loose the dependency to the `perkakas`.
